src/ipc/mem/Segment.cc

```diff
@@ -11,6 +11,8 @@
 #include "squid.h"
 #include "base/TextException.h"
 #include "compat/shm.h"
+#include "compat/socket.h"
+#include "compat/unistd.h"
```

Comment:
With this the redundant `unistd.h` include below can be removed.

Reply:
done